insights/templates/components/footer.html.j2

```diff
@@ -29,7 +31,7 @@
                 <h3 class="footer__heading">Links</h3>
                 <ul>
                     <li><a href="/">Home</a></li>
-                    <li><a href="/about/">About</a></li>
+                    <li><a href="https://insights.threesixtygiving.org/about">About</a></li>
```

Comment:
can we use {{ url_for("about") }} here so that we keep it relative to the application / updates with the code definitions